kml/identify-kml-features/README.md

```diff
@@ -1,26 +1,46 @@
-# Identify KML Features
+# Identify KML features
 
-Get clicked features in a KML Layer.
+Show a callout with formatted content for a KML feature.
 
-![](IdentifyKMLFeatures.png)
+![Image of identify KML features](IdentifyKMLFeatures.png)
+
+## Use case
+
+A user may wish to select a KML feature to view relevant information about it.
 
 ## How to use the sample
 
-When the KML layer is done loaded, click on one of the features. A callout should display next to the feature with HTML describing it.
+Click on a feature to identify it. Feature information will be displayed in a callout.
 
-## How it works
+Note: the KML layer used in this sample contains a screen overlay. The screen overlay contains a legend and the logos for NOAA and the NWS. You can't identify the screen overlay.
 
-To identify KML features:
+## How it works
 
-1. Add a `KmlLayer` to the map as an operational layer.
-2. Add a clicked listener to the map view with `mapView.setOnMouseClicked` to get the clicked screen point.
-3. Identify features at the screen location with `mapView.identifyLayerAsync(kmlLayer, screenPoint, tolerance, false)`.
-4. Use the returned future to get the `IdentifyLayerResult`.
-5. Loop through `result.getElements()`, looking for `GeoElements` which are instances of `KmlPlacemark`.
+1. Listen to clicks on the `MapView` using `setOnMouseClicked()`.
+2. When the map view is clicked:
+  * Dismiss the `Callout`, if one is showing.
+  * Call `identifyLayerAsync(...)` passing in the `KmlLayer`, screen point and tolerance.
+  * Await the result of the identify and then get the `KmlPlacemark` from the result.
+  * Create a callout at the calculated map point and populate the callout content with text from the placemark's `BalloonContent`. NOTE: KML supports defining HTML for balloon content and may need to be converted from HTML to text.
```

Comment:
`BalloonContent` is not a class in our API: however in our sample we call `getBalloonContent` on a KmlNode which returns a string. So I think the following small change is more accurate: 
```suggestion
  * Create a callout at the calculated map point and populate the callout content with text from the placemark's "balloon content". NOTE: KML supports defining HTML for balloon content and may need to be converted from HTML to text.
```